.github/dependabot.yml

```diff
@@ -15,7 +15,11 @@ updates:
         update-types:
           - "minor"
           - "patch"
-
+      prod-dependencies:
+        dependency-type: "production"
```

Comment:
@deivid-rodriguez any recommendations for improving our grouping of Ruby PRs? I'd love to have as many things grouped together as possible.